docker/nvidia/docker-compose.template.yml

```diff
@@ -0,0 +1,10 @@
+version: '2'
+services:
+  dtk_dev:
+    extends:
+      file: ../docker-compose.yml
+      service: dtk_dev
+    image: dtk_dev_cuda
```

Comment:
If you specify `build` it will tag the resulting image in accordance so you want to come up with a different name.